pages/index.tsx

```diff
@@ -1,6 +1,47 @@
 import * as React from "react";
 import IndexPage from "../pageComponents/IndexPage";
+import {
+  NUMBER_OF_POSTS,
+  POPULAR_POSTS_TERM_FROM_IN_MONTH,
+} from "../constants/post";
+import { createGetPopularPosts } from "../repositories/getPopularPosts";
+import postToJSON from "../serializers/postToJSON";
+import getFirebaseApp from "../utilities/getFirebaseApp";
+import jsonToPost from "../serializers/jsonToPost";
 
-export default function Page() {
-  return <IndexPage />;
+export default function Page({
+  prefetchedPopularPostsJSON,
+}: {
+  prefetchedPopularPostsJSON: Record<string, any>[];
+}) {
+  return (
+    <IndexPage
+      prefetchedPopularPosts={prefetchedPopularPostsJSON.map(
+        (prefetchedPopularPostJSON) => jsonToPost(prefetchedPopularPostJSON)
+      )}
```

Comment:
`prefetchedPopularPostsJSON` -> `prefetchedPopularPostJSON` sounds weird for me. Because I think no one says `Children Enrollment` -> `Child Enrollment`

```suggestion
      prefetchedPopularPosts={prefetchedPopularPostsJSON.map((item) => jsonToPost(item))}
```